resident-services/scripts/Resident_Test_script.jmx

```diff
@@ -4464,6 +4464,774 @@ vars.put(&quot;utcTime&quot;, utcTime);</stringProp>
           <hashTree/>
         </hashTree>
       </hashTree>
```

Comment:
channel_type_list_resident.txt is not in this PR, for channel, we might have only 3 or 4 otp channels, can we take is comma separated from user variable itself ?

Reply:
Have added the above file also. And if there will be more otp channels, we can modify the file as required.